engine/src/main/java/org/hibernate/search/analyzer/definition/impl/LuceneTokenizerDefinitionContextImpl.java

```diff
@@ -17,6 +17,10 @@
  */
 public class LuceneTokenizerDefinitionContextImpl implements LuceneAnalysisDefinitionBuilder<TokenizerDef> {
 
+	private static final String PARAMS2 = "params";
```

Comment:
Any reason to use the name `PARAMS2` instead of just `PARAMS`? Maybe you used an automated tool?